src/util/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+add_library(bitcoin_util STATIC EXCLUDE_FROM_ALL
```

Comment:
The check that creates `ENABLE_TRACING` is missing.

That means that tracing support is quietly missing after this PR. I'm a little uncomfortable hooking up code with random defines missing because I'm concerned that we'll forget to hook them up in the end.

Thoughts on either adding the tracing check as part of this PR or somehow documenting the defines that aren't hooked up yet?

Reply:
> The check that creates `ENABLE_TRACING` is missing.

It should be a part of a commit which adds CMake's analogue of `--enable-usdt` configure option, like https://github.com/bitcoin/bitcoin/commit/f4ebea9a1475c9ac7804e8986f986cd528b6d39e.